go/oasis-node/cmd/ias/proxy.go

```diff
@@ -99,8 +99,22 @@ func doProxy(cmd *cobra.Command, _ []string) {
 		return
 	}
 
-	_, tlsKeyPath := TLSCertPaths(dataDir)
+	tlsCertPath, tlsKeyPath := TLSCertPaths(dataDir)
 	cert, err := tlsCert.LoadFromKey(tlsKeyPath, iasProxy.CommonName)
+	if err != nil {
```

Comment:
Is there a more specific error this could match on? so that this only happens in case the certificate doesn't actually exist

Reply:
Fixed, please take another look.

Reply:
:+1: